.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-  - "2.0.0"
+  - 2.2.5
```

Comment:
I'd rather not bump the ruby version. We should be able to tie the version of `rack` and other gems to keep buy at 2.0.0